wine-bash

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+flatpak run --env=WINEPREFIX="${HOME}/.local/share/femm42-flatpak" --command=wine info.femm.unofficial $@
```

Comment:
I don't think this does what you think it should be doing.

`flatpak run` inside the sandbox is wrong.

Reply:
It will run outside.
I send this file to user folder using femm.sh
```cp -f /app/wine-bash ${WINEPREFIX}/wine```